proto/money/rpc/master_folio_svc.proto

```diff
@@ -48,10 +49,23 @@ enum MasterFolioSvcFolioSwapResult {
 	MASTER_FOLIO_SVC_FOLIO_SWAP_FAILURE_UNKNOWN = 3;
 }
 
+message MasterFolioSvcReservationMasterFolioAssignmentList {
+	repeated ReservationMasterFolioAssignment assignments = 1;
+}
+
+message ReservationMasterFolioAssignment {
+	.holms.types.booking.indicators.ReservationIndicator client_folio = 1;
+	.holms.types.booking.indicators.ReservationIndicator master_folio = 2;
+}
+
 message MasterFolioSvcFolioSwapResponse {
 	MasterFolioSvcFolioSwapResult result = 1;
 }
 
+message MasterFolioSvcGetMasterFolioAssignmentsRequest {
+	.holms.types.booking.reservations.ReservationQueryFilter filter = 1;
```

Comment:
This isn't the correct search parameter. It should be a repeated field of reservation indicators:

1) This isn't the booking context, so we shouldn't be porting booking context concepts into here
2) The reservations display has changed significantly so there are a multitude of ways reservations get queried, not to mention the reservation search.